src/App.vue

```diff
@@ -1,12 +1,22 @@
 <template>
   <div id="app">
-    <Header />
+    <Header :tabs="tabs"/>
     <router-view></router-view>
   </div>
 </template>
 
 <script lang="ts" setup>
 import Header from './presentation/components/Header.vue';
+import {IconPicture} from "@codexteam/icons";
+const tabs = [
```

Comment:
I'd suggest moving this to the Header component

Reply:
done